distributed/cli/tests/test_dask_worker.py

```diff
@@ -340,3 +340,48 @@ def test_bokeh_deprecation():
                 await asyncio.sleep(0.01)
             [ws] = s.workers.values()
             assert ws.name == 123
+
+
+WORKER_CLASS_TEXT = """
+from distributed.worker import Worker
+
+class MyWorker(Worker):
+    pass
+"""
```

Comment:
I would prefer moving this within the test function if that's OK.  Or are you trying to avoid indentation issues?

Reply:
Nope, moving into the test function body seems reasonable